README.md

```diff
@@ -46,14 +46,27 @@
-part of the [core
-distribution](https://github.com/open-telemetry/opentelemetry-collector) of the
-Collector. Typically, these contributions are vendor specific
-receivers/exporters and/or components that are only useful to a relatively
-small number of users.
-
-> Please note that this repository and its releases are a superset of the core repository.
+This is a repository for OpenTelemetry Collector components that are not suitable for the  [core repository](https://github.com/open-telemetry/opentelemetry-collector) of the collector. Some of these components are part of the core distribution, such as the Jaeger and Prometheus ones, but most of the components here are only available as part of the "contrib" distribution. Users of the OpenTelemetry Collector are also encouraged to build their own custom distributions with the [OpenTelemetry Collector Builder](https://github.com/open-telemetry/opentelemetry-collector/tree/main/cmd/builder), using the components they need from the core repository, the contrib repository, and possibly third-party or internal repositories.
+
+The official distributions, core and contrib, are available as part of the [opentelemetry-collector-releases](https://github.com/open-telemetry/opentelemetry-collector-releases) repository.
+
+Each component has its own stability and support levels, as defined in the following sections.
+
+## Stability levels
+
+While we intend to provide high-quality components as part of this repository, we acknowledge that not all of them are ready for prime time. As such, each component should list its current stability level, according to the following definitions:
+
+* **In development**: not all pieces of the component are in place yet and it might not be available as part of any distributions yet. Bugs and performance issues should be reported, but it is likely that the component owners might not give them much attention. Your feedback is still desired, especially when it comes to the user-experience (configuration options, component observability, technical implementation details, ...). Configuration options might break often depending on how things evolve. The component should not be used in production.
+* **Experimental**: the component is ready to be used for limited non-critical workloads and the authors of this component would welcome your feedback. Bugs and performance problems should be reported, but component owners might not work on them right away. The configuration options might change between releases, although component owners should try to minimize breaking changes.
+* **Stable**: the component is ready to be used in production. Bugs and performance problems should be reported and there's an expectation that the component owners will work on them. Breaking changes, including configuration options and the component's output are not expected to happen without prior notice, unless under special circumstances.
```

Comment:
Do we need a beta level as well, or RC?

Reply:
How would you describe them both? In my view, experimental encompasses both, and component maintainers would be free to move from experimental to stable as they see fit.

Reply:
I would read the alpha stage as "Experimental" here, while their beta would map to "Stable". I don't think the beta would be a stage in between, given the definitions from this PR.

Reply:
Perhaps it would be better to rename "Experimental" to "Beta"?

Reply:
Maybe breaking changes to the configuration interface can be used as one of differentiator between Experimental and Beta? Current configuration guarantees for Experimental seem too strict IMO.

E.g. **Beta** is ready for production use but the configuration options might change between releases, although component owners should try to minimize breaking changes.

While **Experimental** configuration options might break often depending on how things evolve.

Reply:
Alright, I renamed Experimental to Alpha and added a Beta. As a user, I still think I would prefer to have a single "yes/no" on whether I should use it in production, as the message here is now:

- yes, if it's stable
- maybe, if it's beta (your config might break)
- no, if it's alpha